src/plugins/api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "api",
```

Comment:
If the intention is to tie these api's to kibana then I think they should go into the kibana plugin.

Also, the convention is `/api/{plugin}` not `/{plugin}/api`, does the marvel team have it backwards?